src/kboot.c

```diff
@@ -286,6 +286,52 @@ static int dt_set_mac_addresses(void)
     return 0;
 }
 
+/* adjust PCI port properties, Required for adjusting max-link-speed
+ * for the 10 gigabit ehternet Mac Mini variant.
+ */
+static int dt_adjust_pcie_port_properties(void)
+{
+    int anode = adt_path_offset(adt, "/arm-io/apcie");
+
+    if (anode < 0)
+        bail("ADT: /arm-io/apcie not found\n");
+
+    ADT_FOREACH_CHILD(adt, anode)
+    {
+        const char *anode_name = adt_get_name(adt, anode);
+        if (strncmp(anode_name, "pci-bridge", 10) || strlen(anode_name) != 11)
+            continue;
+
+        char port_num = anode_name[10];
```

Comment:
Probably better to use the `apcie-port` prop here, instead of parsing the node name.

Reply:
agreed, I'll update the change